flocker/node/agents/ebs.py

```diff
@@ -276,6 +278,14 @@ class AttachFailed(Exception):
     """
 
 
+class DetachFailed(Exception):
+    """
+    A volume could not be detached, due to being in busy state.
+    """
```

Comment:
Given the description, please rename this exception to `VolumeBusy` to reflect the cause, and to be consistent with the other exceptions raised by detach.